### PR TITLE
Feat/change creator id trip

### PR DESCRIPTION
**Pull request by g0g05arui**
## Summary

Make creator auto-populated with userID and display only the creator's trips in the overview screen

## Changes

- **Repositories:**: Make `getTrip` take a string argument which is the creatorsID

## Checklist

Ensure all / most of these are checked before the pull request is submitted.
- [x] Tests have been modified for new functionality.

##  Testing
- **Manual Testing:**
    - [x] Tested on emulator / physical device.
- **Unit Tests**
    - [x] Modified units tests for this BUT I need to also mock the Firebase auth somehow ( I will create another ticket for this )

### CODE REVIEW

File: app/src/test/java/com/android/voyageur/model/trip/TripsViewModelTest.kt [hunk 1]
```diff
@@ -49,8 +49,9 @@ class TripsViewModelTest {
 
   @Test
   fun getTripsCallsRepository() {
-    tripsViewModel.getTrips()
-    verify(tripsRepository).getTrips(any(), any())
+
+    // Simulate a successful result from the repository
```
Comment (g0g05arui) [inline app/src/test/java/com/android/voyageur/model/trip/TripsViewModelTest.kt:53]: I will redo this part when we can mock the firebase auth

**Comment by apana03** [comment]
Changes look good, congrats! But please make sure to create an issue for the completion of the unit Tests as we need to make sure to achieve 80% line coverage. Also even though the changes to UI are minimal (removal of a button) make sure to add a screenshot to the PR.

File: app/src/test/java/com/android/voyageur/model/trip/TripsViewModelTest.kt (hunk 1 above)
Comment (apana03) [reply app/src/test/java/com/android/voyageur/model/trip/TripsViewModelTest.kt:53]: yes, good idea but do not forget to do so please :)

**Comment by g0g05arui** [comment]
>  Also even though the changes to UI are minimal (removal of a button) make sure to add a screenshot to the PR.
Yes, great idea. From now on we should keep SS in mind, thanks for the suggestion.

**Pull request merged**